src/index-page.css

```diff
@@ -0,0 +1,46 @@
+/*button on apply page*/
+.button-index {
+    display: inline-block;
+    /*text-decoration: none;
```

Comment:
the comments that are descriptive are okay, however these lines of code in your ```.button-index``` class need to be deleted if you aren't using them!